src/components/views/messages/DateSeparator.tsx

```diff
@@ -132,23 +132,67 @@ export default class DateSeparator extends React.Component<IProps, IState> {
                     `found ${eventId} (${originServerTs}) for timestamp=${unixTimestamp} (looking forward)`,
             );
 
-            dis.dispatch<ViewRoomPayload>({
-                action: Action.ViewRoom,
-                event_id: eventId,
-                highlighted: true,
-                room_id: roomId,
-                metricsTrigger: undefined, // room doesn't change
-            });
-        } catch (e) {
-            const code = e.errcode || e.statusCode;
-            // only show the dialog if failing for something other than a network error
-            // (e.g. no errcode or statusCode) as in that case the redactions end up in the
-            // detached queue and we show the room status bar to allow retry
```

Comment:
This comment was bogus. Just copy/paste cruft from 

https://github.com/matrix-org/matrix-react-sdk/blob/bc60e59eda904fd35a0b7c3f5b6a041114e45034/src/components/views/dialogs/ConfirmRedactDialog.tsx#L96-L98